vlib/v/help/build/build.txt

```diff
@@ -159,7 +159,7 @@ NB: the build flags are shared with the run command too:
     Note: the same effect can be achieved by setting the environment variable VNORUN to 1.
 
   -skip-unused
-    Skip generating C/JS code for functions, that are provably not used by your project.
+    Skip generating C/JS code for functions, that are probably not used by your project.
```

Comment:
That is not a typo. What -skip-unused does is *prove* that certain features are used, and then marks them as such. There is nothing probabilistic about its action.